File: app/src/main/java/com/doludolu/baseproject/mode/javabean/base/AppUpdata.java
package com.doludolu.baseproject.mode.javabean.base;

/**
 * Created by Administrator on 2016/6/20.
 */
public class AppUpdata {
    /**
     * ID 		int		自增ID
     * VersionNum 	int		版本号
     * VersionName	varchar(50)	版本名
     * UpdateTiime	datetime	更新时间
     * <p/>
     * AppPath		varchar(100)	路径
     * IsUPdate                     是否强制升级
     * VersionIntro    nvarchar(200)   版本介绍
     */
    private boolean IsUPdate;
    private int VersionNum;
    private String AppPath;
    private String VersionIntro;
    private String VersionName;

    public boolean isUPdate() {
        return IsUPdate;
    }

    public void setIsUPdate(boolean UPdate) {
        IsUPdate = UPdate;
    }

    public int getVersionNum() {
        return VersionNum;
    }

    public void setVersionNum(int versionNum) {
        VersionNum = versionNum;
    }

    public String getAppPath() {
        return AppPath;
    }

    public void setAppPath(String appPath) {
        AppPath = appPath;
    }

    public String getVersionIntro() {
        return VersionIntro;
    }

    public void setVersionIntro(String versionIntro) {
        VersionIntro = versionIntro;
    }

    public String getVersionName() {
        return VersionName;
    }

    public void setVersionName(String versionName) {
        VersionName = versionName;
    }
}

File: libcore/src/main/java/com/ashlikun/libcore/javabean/AppUpdata.kt
package com.ashlikun.libcore.javabean

/**
 * @author　　: 李坤
 * 创建时间: 2018/7/18 13:17
 * 邮箱　　：496546144@qq.com
 *
 *
 * 功能介绍：app更新的实体类
 */

class AppUpdata {
    /**
     * ID 		int		自增ID
     * VersionNum 	int		版本号
     * VersionName	varchar(50)	版本名
     * UpdateTiime	datetime	更新时间
     *
     *
     * AppPath		varchar(100)	路径
     * IsUPdate                     是否强制升级
     * VersionIntro    nvarchar(200)   版本介绍
     */
    var IsUPdate = false
    var VersionNum = 0
    var AppPath: String? = null
    var VersionIntro: String? = null
    var VersionName: String? = null


}
